packages/ndla-notion/src/LicenseByline/LicenseDescription.tsx

```diff
@@ -9,10 +9,10 @@
 import { ReactNode } from "react";
 import styled from "@emotion/styled";
 import { colors, spacing } from "@ndla/core";
-import LicenseIconDescriptionList from "./LicenseIconDescriptionList";
+import LicenseIconDescriptionList from "./LicenseDescriptionList";
```

Comment:
Lurt å oppdatere importen.
```suggestion
import LicenseDescriptionList from "./LicenseDescriptionList";
```